grails-app/i18n/messages.properties

```diff
@@ -1300,7 +1300,12 @@ kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCo
 kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.districtId.modal.button.ok = Yes, delete all the votes and supports
 kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.districtId.modal.button.cancel = Ouch, no. Let it as is.
 kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.cause.label = Cause
+kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.backerType.label = Type
+kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.backerType.CITIZEN = Citizen
+kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.backerType.PUBLIC_INSTITUTION = Public Institution
+kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.backerType.CITIZEN_INSTITUTION = Citizen Institution
```

Comment:
If you use the formUtil taglib, this messages will not match the prefix will be the package of BackerType